src/ariel-os/Cargo.toml

```
@@ -160,6 +160,21 @@ ble-peripheral = ["ble", "ariel-os-embassy/ble-peripheral"]
 # Enables BLE to act as a central. Can be used in conjunction with `ble-peripheral`.
 ble-central = ["ble", "ariel-os-embassy/ble-central"]
 
+#! ## Timer queue size configuration
+#! Should only be enabled when threading is enabled, too.
+#! Default is 64 entries for regular boards and 8 entries for those with little ram.
+#! While more than this number of concurrent timers are used, the system will loop (and not sleep/idle anymore).
```

Could this be clarified further? Does "loop"ing means busy looping? When exactly during runtime would it loop?
```suggestion
#! When more than this number of concurrent timers are used, the system will loop (and not sleep/idle anymore).
```

---

IIUC when no free slot can be found, a random `Waker` is popped and awoken. So the executor(s) and thread(s) for that waker would `poll()` that worker instead of waiting for the timer.

What wording would make that clearer?